packer/ansible/rackhd_source.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Development Server
+  hosts: local
+  sudo: no
+  roles:
+    - apt
+    - build
+    - git
+    - rabbitmq
+    - mongodb
+    - node
+    - isc-dhcp-server
+    - monorail
```

Comment:
This playbook was modeled on the rackhd_local.yml playbook:
https://github.com/RackHD/RackHD/blob/master/packer/ansible/rackhd_local.yml